packages/gatsby-source-filesystem/src/create-remote-file-node.js

```diff
@@ -46,17 +46,7 @@ let showFlagWarning = !!process.env.GATSBY_EXPERIMENTAL_REMOTE_FILE_PLACEHOLDER
  * Queue Management *
  ********************/
 
-/**
- * Queue
- * Use the task's url as the id
- * When pushing a task with a similar id, prefer the original task
- * as it's already in the processing cache
- */
-const queue = new Queue(pushToQueue, {
-  id: `url`,
-  merge: (old, _, cb) => cb(old),
-  concurrent: process.env.GATSBY_CONCURRENT_DOWNLOAD || 200,
-})
```

Comment:
We already protect ourselves from multiple jobs for the same URL here (so `merge` option is redundant):

https://github.com/gatsbyjs/gatsby/blob/d7d08508d3eb88eff87867fc7121c617cd02ce8e/packages/gatsby-source-filesystem/src/create-remote-file-node.js#L244-L248